wiki/Tournaments/VAF/1/en.md

```diff
@@ -0,0 +1,326 @@
+| HiroK ![][flag_BR] | 2 | **7** | ![][flag_BR] **Gustav0eiji** | [#1](https://osu.ppy.sh/community/matches/83413631), [#2](https://osu.ppy.sh/community/matches/83416800) |
+| **D3kuu** ![][flag_IT] | **7** | 1 | ![][flag_MY] HoopWheat | [#1](https://osu.ppy.sh/community/matches/83417440) |
+| **Skull Kid** ![][flag_BR] | **7** | 2 | ![][flag_CH] MC2BP | [#1](https://osu.ppy.sh/community/matches/83418728) |
+| **\_knots** ![][flag_US] | **7** | 4 | ![][flag_BE] XOlifreX | [#1](https://osu.ppy.sh/community/matches/83422824) |
+| **LordEnder** ![][flag_IT] | **7** | 2 | ![][flag_BR] FurryBoujee | [#1](https://osu.ppy.sh/community/matches/83424964) |
+
+Sunday, 16 May 2021:
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Ppus** ![][flag_BR] | **7** | 2 | ![][flag_BR] Coryn | [#1](https://osu.ppy.sh/community/matches/83446275) |
+| **janitore** ![][flag_CA] | **7** | 0 | ![][flag_MX] -Arato | [#1](https://osu.ppy.sh/community/matches/83452492) |
+| **\[Zeth\]** ![][flag_MY] | **7** | 3 | ![][flag_US] Garpo | [#1](https://osu.ppy.sh/community/matches/83458473) |
+| r1chyy ![][flag_AU] | 3 | **7** | ![][flag_JP] **Ku\_Ren** | [#1](https://osu.ppy.sh/community/matches/83464262) |
+| phosphene47 ![][flag_KR] | 4 | **7** | ![][flag_JP] **Saikoro** | [#1](https://osu.ppy.sh/community/matches/83468725) |
+| **XK2238** ![][flag_JP] | **7** | 1 | ![][flag_TH] Ponamis | [#1](https://osu.ppy.sh/community/matches/83473055) |
+| **Minekuchi** ![][flag_DE] | **7** | 3 | ![][flag_JP] shakegohan\_jar | [#1](https://osu.ppy.sh/community/matches/83473056) |
+| **oL0** ![][flag_KR] | **7** | 0 | ![][flag_RU] CrowWhite | [#1](https://osu.ppy.sh/community/matches/83475194) |
+| **Akonine** ![][flag_RU] | **7** | 2 | ![][flag_JP] uranosu1030 | [#1](https://osu.ppy.sh/community/matches/83477641) |
+| **Ikkun** ![][flag_IT] | **7** | 0 | ![][flag_RU] Den4ik228 | [#1](https://osu.ppy.sh/community/matches/83477972) |
+| **5henry** ![][flag_KR] | **7** | 1 | ![][flag_ID] Naverlyn | [#1](https://osu.ppy.sh/community/matches/83479482) |
+| petterde ![][flag_NO] | 4 | **7** | ![][flag_KR] **Jakads** | [#1](https://osu.ppy.sh/community/matches/83484317) |
+| **Genjuro** ![][flag_GR] | **7** | 1 | ![][flag_TW] \_Llama | [#1](https://osu.ppy.sh/community/matches/83488630) |
+| **coed** ![][flag_IT] | **0** | -1 | ![][flag_FR] BananaW | *win by default* |
+| **cheese salad** ![][flag_US] | **7** | 0 | ![][flag_GB] Teezel | [#1](https://osu.ppy.sh/community/matches/83500633) |
+| **Vulkin** ![][flag_CL] | **7** | 3 | ![][flag_NL] Cookie\_tree | [#1](https://osu.ppy.sh/community/matches/83500624) |
+| **gaston\_2199** ![][flag_AR] | **7** | 0 | ![][flag_FI] MEGAMELA | [#1](https://osu.ppy.sh/community/matches/83504892) |
+| **AuroraPhasmata** ![][flag_US] | **7** | 4 | ![][flag_US] Etholux | [#1](https://osu.ppy.sh/community/matches/83519527) |
+
+## Ruleset
+
+### General rules
```

Comment:
this section needs to be proofread, could you please give it one and fix things like `At the time of the match. Players will be invited by the referee.`, `noshowing`, or `Highest `will be ... asked to pick one` yourself? I'll recheck it afterwards

Reply:
Will do alongside new updates tomorrow, since it'll be new stage by Monday.

Reply:
do you mean that you prefer this to be merged as soon as possible, or that you will fix it tomorrow?

Reply:
Will proofread it by tomorrow after having new commits. So this can hold on with no problem.

Reply:
*Will fix it by tomorrow